site/content/docs/5.1/examples/title-bars/index.html

```diff
@@ -0,0 +1,53 @@
+---
+layout: examples
+title: Title Bars Template
+extra_css:
+  - "title-bars.css"
+html_class: "h-100"
+body_class: "d-flex flex-column"
+---
+
+<main>
+  <div class="alert alert-info mb-0" role="alert">
+    <span class="alert-icon"><span class="visually-hidden">Information</span></span>
+    <div>
+      <h4 class="alert-heading">Background image</h4>
+      <p>In order to have a correct background image in title bars, an extra class and some CSS are required. The corresponding class is prefixed with <code>.example-title-bar-custom-</code> and can be found in <a href="title-bars.css" title="Title bars example CSS file">title-bars.css</a>.</p>
```

Comment:
IMO this text can be improved to explain that all three examples are not using the same "image system" and that only the blue one (the 2nd) needs an extra class named `.example-title-bar-custom-background` and an extra CSS definition.